CHANGELOG-1.2.md

```diff
@@ -0,0 +1,92 @@
+# Release notes for 1.2.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.1.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Fix a bug that when CSI migration is enabled and PV is using GA topology label, resizing is not work. ([#139](https://github.com/kubernetes-csi/external-resizer/pull/139), [@Jiawei0227](https://github.com/Jiawei0227))
+
+### Feature
+
+- Add a feature-gate to allow users to restore size of deleted PVCs ([#140](https://github.com/kubernetes-csi/external-resizer/pull/140), [@sunpa93](https://github.com/sunpa93))
+
+### Bug fixes
+
+- Set the value of "migrated" field in the metrics to true or false to indicate if the call is a migration enabled feature or not ([#138](https://github.com/kubernetes-csi/external-resizer/pull/138), [@nearora-msft](https://github.com/nearora-msft))
```

Comment:
Can we copy the wording that we used for the other sidecars?

Reply:
done